src/components/molecules/navbar/navbarMolecule.css

```diff
@@ -1,5 +1,12 @@
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css");
```

Comment:
Just a suggestion: wouldn't it be better to install this Bootstrap's depency too instead of using cdn? 
Anyway, we need to deploy our project ASAP, so I won't hold back your work because of this.

Reply:
The change is already made. Very good observation. thank you